src/__snapshots__/footer.test.jsx.snap

```diff
@@ -132,72 +132,72 @@ exports[`Footer exists 1`] = `
                     ]
                   }
                 />
-              </FlexItem>
-            </Flex>
-          </FlexItem>
-          <FlexItem
+              </Component>
+            </Component>
+          </Component>
+          <Component
             shrink={true}
           >
-            <Flex
+            <Component
               direction="column"
               noGutters={true}
             >
-              <FlexItem
+              <Component
                 shrink={true}
               >
-                <Chunk
+                <Component
                   className="align--center atMedium_align--left"
                 >
                   <SocialIconsList
                     localeCode="en-US"
                   />
-                </Chunk>
-              </FlexItem>
-              <FlexItem>
-                <Flex
+                </Component>
+              </Component>
+              <Component>
+                <Component
                   direction="column"
                   justify={null}
                 >
-                  <FlexItem
+                  <Component
                     className="margin--top"
                     shrink={true}
                   >
-                    <Chunk>
+                    <Component>
                       <LanguageSelectInput
                         currentLocaleCode="en-US"
                         onChange={[Function]}
                       />
-                    </Chunk>
-                  </FlexItem>
-                  <FlexItem
+                    </Component>
+                  </Component>
+                  <Component
                     shrink={true}
                   >
-                    <Chunk
+                    <Component
                       className="align--center atMedium_align--left margin--top"
                     >
                       <AppBadges
                         getAppStorePhoto={[Function]}
                         language="en"
                       />
-                    </Chunk>
-                  </FlexItem>
-                </Flex>
-              </FlexItem>
-            </Flex>
-          </FlexItem>
-        </Flex>
-      </Section>
-    </Bounds>
-  </Stripe>
-  <Stripe
+                    </Component>
```

Comment:
Looks like the HOC isn't correctly setting the `displayName` of the passed component